File: apps/muly-dotnet-server/src/APIs/Member/Dtos/MemberWhereInput.cs
namespace MulyDotnet.APIs.Dtos;

public class MemberWhereInput
{
    public string? Id { get; set; }

    public DateTime? CreatedAt { get; set; }

    public DateTime? UpdatedAt { get; set; }

    public DateTime? MembershipDate { get; set; }

    public string? FirstName { get; set; }

    public string? LastName { get; set; }
}

File: apps/muly-node-js-admin/src/api/member/MemberWhereInput.ts
import { StringFilter } from "../../util/StringFilter";
import { StringNullableFilter } from "../../util/StringNullableFilter";
import { DateTimeNullableFilter } from "../../util/DateTimeNullableFilter";

export type MemberWhereInput = {
  id?: StringFilter;
  firstName?: StringNullableFilter;
  lastName?: StringNullableFilter;
  membershipDate?: DateTimeNullableFilter;
};
